.github/workflows/asst-ci.yml

```diff
@@ -13,136 +16,143 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
+    
+      - name: Test env GITHUB_REF_NAME
+        run: echo "GITHUB_REF_NAME=$GITHUB_REF_NAME"
```

Comment:
If we use github.ref_name as the environment variable for checking the branch name, is line 20 necessary?

Reply:
No it's not necessary. I will remove it.

It was for testing purposes.

Reply:
Awesome.